addon/components/list-card.js

```diff
@@ -248,3 +287,6 @@ export default Ember.Component.extend({
     }
   }
 });
+
+export default ListCardComponent;
+export { ListCardComponent, QueryToken, QueryFilter, FilterGroup };
```

Comment:
This is cool - do you know if this is best practice over having each of these in their own file?